### ISSUE TITLE
Face detection using a cascade classifier no longer works

### ISSUE THREAD

**Issue by changbaishan** [scikit-image/skimage-demos#7]
# Load the trained file from the module root.
trained_file = data.lbp_frontal_face_cascade_filename()

the data object no longer contains lbp_frontal_face_cascade_filename()

# Initialize the detector cascade.
detector = Cascade(trained_file)

skimage.feature no longer contains  Cascade function.

This example needs to be rewritten.

**Comment by stefanv** [comment]
Yeah, this repo is way out of date.  Would you like to file a PR?

**Comment by changbaishan** [comment]
Thanks for your answer, Stefan. Can you please explain "file an PR"? I am running your examples to learn.

Best regards,

Yong Liu
________________________________

**Comment by stefanv** [comment]
Apologies for the shorthand: PR == pull request, i.e., send us the changes necessary to make this work.  If you're not comfortable making those changes, no problem, we can do it.

I would recommend you look at the tutorials repository instead.  Those are kept up to date, and are used fairly regularly.

**Comment by changbaishan** [comment]
Hi Stefan,

Thanks for your comments.

I tried to do a PR, but unable to proceed. 

Can you please enlighten how I can access the tutorials repository?

Thanks

Yong Liu
